admin-app/dictionary/views.py

```
@@ -0,0 +1,150 @@
+
+
+def _sanitize_query(raw: str) -> str:
+    """Strip whitespace and remove characters that could break query parsing."""
+    text = raw.strip()[:200]  # limit length
+    # Remove characters that are special in query_string syntax (safety net)
+    text = re.sub(r'[+\-=&|><!(){}\[\]^"~*?:\\/]', ' ', text)
+    # Collapse multiple spaces
+    text = re.sub(r'\s+', ' ', text).strip()
+    return text
+
+@require_GET
```

Ruff (0.15.2)

[error] Undefined name `require_GET`

(F821)